Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM alpine:3.12
```

Comment:
Not sure if this will limit executables within the container or if debian slim would be a better option.